### DISCUSSION TITLE
LTN & car-free discussion

### DISCUSSION THREAD

**Discussion by dabreegster** [dabreegster/abstreet#770] [General]
There's lots of good debate and examples flying around #660, but that issue should stay focused on how to render/visualize such areas in A/B Street. Let's chat here instead!

**Comment by westnordost** [comment]
The biggest car-free place in terms of population is the old city ("Medina") of Fez in Morocco. 

OpenStreetMap at this place looks awful https://www.openstreetmap.org/#map=17/34.06396/-4.97260 because most of those tiny alleyways can not be seen from satellite imagery and obviously people are unsure about how to tag it - as footways, as `service=alley` or as pedestrian roads.

I recommend visiting this otherworldly place once, it truly feels like a time travel. Especially the dense center feels like one giant building (with open-air roof).

**Comment by westnordost** [comment]
The issue I see with places like Vauban and other "sustainable" neighbourhoods is that it seems to be just for the "hip" people who can affort it. As long as being green and environment friendly is only afforded by the upper class, it can't work.

The same problem I see with most of the things that are not car-centered. I have posted a few times in OSM slack about the "critical mass" bicycle protest with videos of thousands of cyclists roaming through the streets of Hamburg. People who live further outside in cheap(er) suburbs don't have the luxury to use the bicycle for every-day transportation, so they necessarily see the bicyclist as their enemy. So, basically, many of those protesting are those that can afford to live in the center of town.

Not sure if anything big can change as long as motorcars themselves do not become a luxury.

**Comment by Robinlovelace** [comment]
General comment on this, there are many ways to detect communities and I'm sure A/B Street's current algorithm could do better:

![image](https://user-images.githubusercontent.com/1825120/139701862-ad22becc-4d53-4e99-b6e6-5336502a733a.png)

How to quantify 'better' though? I think with public outreach orgs like Possible onboard we could crowd-source public's definition of neighbourhoods (or where they would want LTNs around where they live?) and then compare different algorithms and parameters in a 'model/parameter space' to see which fit people's intuition best.
